### PR TITLE
facility reporting proof of concept

### PR DESCRIPTION
**Pull request by czue**
proof of concept for facility-level report-routing + cleanup

@gcapalbo review by commit

### CODE REVIEW

File: logistics_project/apps/malawi/warehouse/report_views/reporting_rate.py [hunk 1]
```diff
@@ -18,6 +19,9 @@
 class View(warehouse_view.DistrictOnlyView):
 
     def custom_context(self, request):
+        if request.is_facility:
```
Comment (gcapalbo) [inline logistics_project/apps/malawi/warehouse/report_views/reporting_rate.py]: i don't know how widespread this view class is used but if is_facility doesn't get set for each view then we should change this to `getattr(request, 'is_facility', False)`

**Comment by gcapalbo** [comment]
one comment otherwise :+1:

**Comment by gcapalbo** [comment]
(also, feel free to self-merge if that's nothing to worry about, or once it's addressed)

File: logistics_project/apps/malawi/warehouse/report_views/reporting_rate.py (hunk 1 above)
Comment (czue) [inline logistics_project/apps/malawi/warehouse/report_views/reporting_rate.py]: I'm pretty sure it should always be set so i'm ok failing hard and treating it like a requirement

**Pull request merged**